db/migrate/20170329064703_create_user_emails.rb

```diff
@@ -0,0 +1,26 @@
+class CreateUserEmails < ActiveRecord::Migration
+  def up
+    create_table :user_emails do |t|
+      t.references :user
```

Comment:
we never really use this in migrations, so I am not sure I want to start now... t.integer :user_id + explicit index creation should do the trick.